src/routes/resources/+page.svelte

```diff
@@ -1,219 +1,224 @@
 <script lang="ts">
-    import { onMount } from "svelte";
-    import { github_url } from "$lib/constants";
-    import type { PageData } from "./$types";
-    export let data: PageData;
-
-    let resources = data.payload.resources;
-    let displayedResources = resources;
-    let tagLogicAnd: boolean = true; // Whether all the selected tags must match the resource (vs any of the selected tags)
-    // TODO: make this a user preference
-    $: tagLogic = tagLogicAnd ? "and" : "or";
-
-    let tags = data.payload.tags;
-    // Creating filter object
-    let filterObject: any = {};
-    filterObject["tags"] = {};
+  import { onMount } from "svelte";
```

Comment:
Did you change the tabbing here? I'm struggling a bit to see the diff. it looks like maybe you moved something up in the logic. May I suggest that you revert this tab change and limit the change to just the added logic. Is there a standard for how we format these svelte files. We may want to add some automation task to check prettier if not and make all those changes at once.